src/prisma/generator/templates/client.py.jinja

```diff
@@ -150,7 +150,19 @@ class Prisma:
         self._active_provider = '{{ active_provider }}'
         self._log_queries = log_queries
         self._datasource = datasource
-        self._connect_timeout = connect_timeout
+        if connect_timeout is None or isinstance(connect_timeout, UseClientDefault):
+            self._connect_timeout = datetime.timedelta(seconds=10)
+        elif isinstance(connect_timeout, int):
+            message = (
+                'Using integers for connect_timeout argument is deprecated'
+                ' and will be removed in the next release.\n'
```

Comment:
```suggestion
                ' and will be removed in the next major release.\n'
```

This would be a breaking change so I am going to hold off on enforcing this deprecation until the next major release.